fsharp-backend/tests/Tests/DvalRepr.Tests.fs

```diff
@@ -107,6 +107,18 @@ let testDateMigrationHasCorrectFormats =
     Expect.equal newActual $"\"{str}\"" "old format"
   }
 
+// We used a System.Text.Json converter supplied by a NuGet package for a bit,
+// but found that it was incompatible with the OCamlCompatible serializer. We
+// have since adjusted `Vanilla` to use a custom converter, and this test is to
+// ensure values serialized during the time where the NuGet package's converter
+// are able to be deserialized. The value here
+let testPreviousDateSerializionCompatibility =
+  test "previous date serialization compatible" {
+    let expected = RT.DDate(NodaTime.Instant.UnixEpoch.toUtcLocalTimeZone ())
+    let actual =
+      Json.Vanilla.deserialize<RT.Dval> """["DDate","1970-01-01T00:00:00"]"""
+    Expect.equal expected actual "not deserializing correctly"
+  }
```

Comment:
I wasn't feeling very creative about naming this test! feels a bit meh, though maybe I'm nitpicky at myself here.

Reply:
better than `DvalReprInternal'`